[2023-12-30T11:30:33.268+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_silver_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T11:30:33.281+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_silver_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T11:30:33.281+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T11:30:33.281+0000] {taskinstance.py:1363} INFO - Starting attempt 6 of 6
[2023-12-30T11:30:33.281+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T11:30:33.294+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_silver_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T11:30:33.299+0000] {standard_task_runner.py:55} INFO - Started process 7629 to run task
[2023-12-30T11:30:33.301+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_silver_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmpngw047fr']
[2023-12-30T11:30:33.302+0000] {standard_task_runner.py:83} INFO - Job 125: Subtask run_dbt_silver_models
[2023-12-30T11:30:33.344+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_silver_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T11:30:33.391+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_silver_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T11:30:33.392+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T11:30:33.392+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select silver"]
[2023-12-30T11:30:33.400+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T11:30:34.976+0000] {subprocess.py:93} INFO - [0m11:30:34  Running with dbt=1.4.6
[2023-12-30T11:30:35.064+0000] {subprocess.py:93} INFO - [0m11:30:35  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T11:30:35.066+0000] {subprocess.py:93} INFO - [0m11:30:35
[2023-12-30T11:30:35.144+0000] {subprocess.py:93} INFO - [0m11:30:35  Concurrency: 4 threads (target='prod_***')
[2023-12-30T11:30:35.144+0000] {subprocess.py:93} INFO - [0m11:30:35
[2023-12-30T11:30:35.157+0000] {subprocess.py:93} INFO - [0m11:30:35  1 of 3 START sql incremental model silver.customer_information ................. [RUN]
[2023-12-30T11:30:35.161+0000] {subprocess.py:93} INFO - [0m11:30:35  2 of 3 START sql incremental model silver.daily_customer_drivers ............... [RUN]
[2023-12-30T11:30:35.162+0000] {subprocess.py:93} INFO - [0m11:30:35  3 of 3 START sql incremental model silver.loan_transactions .................... [RUN]
[2023-12-30T11:30:35.268+0000] {subprocess.py:93} INFO - [0m11:30:35  2 of 3 ERROR creating sql incremental model silver.daily_customer_drivers ...... [[31mERROR[0m in 0.10s]
[2023-12-30T11:30:35.428+0000] {subprocess.py:93} INFO - [0m11:30:35  3 of 3 OK created sql incremental model silver.loan_transactions ............... [[32mINSERT 0 9613[0m in 0.26s]
[2023-12-30T11:30:35.447+0000] {subprocess.py:93} INFO - [0m11:30:35  1 of 3 OK created sql incremental model silver.customer_information ............ [[32mINSERT 0 28497[0m in 0.28s]
[2023-12-30T11:30:35.478+0000] {subprocess.py:93} INFO - [0m11:30:35
[2023-12-30T11:30:35.478+0000] {subprocess.py:93} INFO - [0m11:30:35  Finished running 3 incremental models in 0 hours 0 minutes and 0.41 seconds (0.41s).
[2023-12-30T11:30:35.488+0000] {subprocess.py:93} INFO - [0m11:30:35
[2023-12-30T11:30:35.489+0000] {subprocess.py:93} INFO - [0m11:30:35  [31mCompleted with 1 error and 0 warnings:[0m
[2023-12-30T11:30:35.489+0000] {subprocess.py:93} INFO - [0m11:30:35
[2023-12-30T11:30:35.489+0000] {subprocess.py:93} INFO - [0m11:30:35  [33mDatabase Error in model daily_customer_drivers (models/silver/daily_customer_drivers.sql)[0m
[2023-12-30T11:30:35.490+0000] {subprocess.py:93} INFO - [0m11:30:35    operator does not exist: text < integer
[2023-12-30T11:30:35.490+0000] {subprocess.py:93} INFO - [0m11:30:35    LINE 22:         WHEN health_score < 100 THEN 100
[2023-12-30T11:30:35.491+0000] {subprocess.py:93} INFO - [0m11:30:35                                       ^
[2023-12-30T11:30:35.491+0000] {subprocess.py:93} INFO - [0m11:30:35    HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
[2023-12-30T11:30:35.491+0000] {subprocess.py:93} INFO - [0m11:30:35    compiled Code at target/run/dwh/models/silver/daily_customer_drivers.sql
[2023-12-30T11:30:35.491+0000] {subprocess.py:93} INFO - [0m11:30:35
[2023-12-30T11:30:35.491+0000] {subprocess.py:93} INFO - [0m11:30:35  Done. PASS=2 WARN=0 ERROR=1 SKIP=0 TOTAL=3
[2023-12-30T11:30:36.387+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-12-30T11:30:36.396+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-12-30T11:30:36.398+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=dbt_model, task_id=run_dbt_silver_models, execution_date=20220910T000000, start_date=20231230T113033, end_date=20231230T113036
[2023-12-30T11:30:36.411+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 125 for task run_dbt_silver_models (Bash command failed. The command returned a non-zero exit code 1.; 7629)
[2023-12-30T11:30:36.447+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-12-30T11:30:36.484+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-30T12:47:23.525+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_silver_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T12:47:23.533+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_silver_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T12:47:23.533+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T12:47:23.533+0000] {taskinstance.py:1363} INFO - Starting attempt 6 of 6
[2023-12-30T12:47:23.533+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T12:47:23.543+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_silver_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T12:47:23.546+0000] {standard_task_runner.py:55} INFO - Started process 11545 to run task
[2023-12-30T12:47:23.549+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_silver_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmpk82st4d7']
[2023-12-30T12:47:23.549+0000] {standard_task_runner.py:83} INFO - Job 133: Subtask run_dbt_silver_models
[2023-12-30T12:47:23.590+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_silver_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T12:47:23.636+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_silver_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T12:47:23.636+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T12:47:23.637+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select silver"]
[2023-12-30T12:47:23.645+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T12:47:25.254+0000] {subprocess.py:93} INFO - [0m12:47:25  Running with dbt=1.4.6
[2023-12-30T12:47:25.346+0000] {subprocess.py:93} INFO - [0m12:47:25  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T12:47:25.348+0000] {subprocess.py:93} INFO - [0m12:47:25
[2023-12-30T12:47:25.414+0000] {subprocess.py:93} INFO - [0m12:47:25  Concurrency: 4 threads (target='prod_***')
[2023-12-30T12:47:25.415+0000] {subprocess.py:93} INFO - [0m12:47:25
[2023-12-30T12:47:25.422+0000] {subprocess.py:93} INFO - [0m12:47:25  1 of 3 START sql incremental model silver.customer_information ................. [RUN]
[2023-12-30T12:47:25.422+0000] {subprocess.py:93} INFO - [0m12:47:25  2 of 3 START sql incremental model silver.daily_customer_drivers ............... [RUN]
[2023-12-30T12:47:25.423+0000] {subprocess.py:93} INFO - [0m12:47:25  3 of 3 START sql incremental model silver.loan_transactions .................... [RUN]
[2023-12-30T12:47:25.542+0000] {subprocess.py:93} INFO - [0m12:47:25  2 of 3 ERROR creating sql incremental model silver.daily_customer_drivers ...... [[31mERROR[0m in 0.12s]
[2023-12-30T12:47:25.728+0000] {subprocess.py:93} INFO - [0m12:47:25  3 of 3 OK created sql incremental model silver.loan_transactions ............... [[32mINSERT 0 9613[0m in 0.30s]
[2023-12-30T12:47:25.743+0000] {subprocess.py:93} INFO - [0m12:47:25  1 of 3 OK created sql incremental model silver.customer_information ............ [[32mINSERT 0 28497[0m in 0.32s]
[2023-12-30T12:47:25.764+0000] {subprocess.py:93} INFO - [0m12:47:25
[2023-12-30T12:47:25.764+0000] {subprocess.py:93} INFO - [0m12:47:25  Finished running 3 incremental models in 0 hours 0 minutes and 0.42 seconds (0.42s).
[2023-12-30T12:47:25.777+0000] {subprocess.py:93} INFO - [0m12:47:25
[2023-12-30T12:47:25.777+0000] {subprocess.py:93} INFO - [0m12:47:25  [31mCompleted with 1 error and 0 warnings:[0m
[2023-12-30T12:47:25.777+0000] {subprocess.py:93} INFO - [0m12:47:25
[2023-12-30T12:47:25.778+0000] {subprocess.py:93} INFO - [0m12:47:25  [33mDatabase Error in model daily_customer_drivers (models/silver/daily_customer_drivers.sql)[0m
[2023-12-30T12:47:25.778+0000] {subprocess.py:93} INFO - [0m12:47:25    operator does not exist: text < integer
[2023-12-30T12:47:25.778+0000] {subprocess.py:93} INFO - [0m12:47:25    LINE 22:         WHEN health_score < 100 THEN 100
[2023-12-30T12:47:25.778+0000] {subprocess.py:93} INFO - [0m12:47:25                                       ^
[2023-12-30T12:47:25.779+0000] {subprocess.py:93} INFO - [0m12:47:25    HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
[2023-12-30T12:47:25.779+0000] {subprocess.py:93} INFO - [0m12:47:25    compiled Code at target/run/dwh/models/silver/daily_customer_drivers.sql
[2023-12-30T12:47:25.779+0000] {subprocess.py:93} INFO - [0m12:47:25
[2023-12-30T12:47:25.779+0000] {subprocess.py:93} INFO - [0m12:47:25  Done. PASS=2 WARN=0 ERROR=1 SKIP=0 TOTAL=3
[2023-12-30T12:47:26.724+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-12-30T12:47:26.734+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-12-30T12:47:26.738+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=dbt_model, task_id=run_dbt_silver_models, execution_date=20220910T000000, start_date=20231230T124723, end_date=20231230T124726
[2023-12-30T12:47:26.753+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 133 for task run_dbt_silver_models (Bash command failed. The command returned a non-zero exit code 1.; 11545)
[2023-12-30T12:47:26.773+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-12-30T12:47:26.803+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
